sql-statements/sql-statement-with.md

```diff
@@ -43,8 +43,6 @@ IdentListWithParenOpt ::=
 
 Non-recursive CTE:
 
-{{< copyable "sql" >}}
-
 ```sql
 WITH CTE AS (SELECT 1, 2) SELECT * FROM cte t1, cte t2;
```

Comment:
maybe use a lowercase `cte` here as it seems you're fixing the casing in other PRs too